src/Components/UserProfile/User.js

```diff
@@ -26,21 +26,23 @@ function User() {
   }, [username])
 
   return (
-    <main>
+    <>
       <Navbar
         start={
           <Link to="/" aria-label="Go back to Home">
             <i className="pi pi-arrow-left"></i>
           </Link>
         }
       />
-      {showProgress && <ProgressBar mode="indeterminate" />}
-      {skeleton && <Placeholder />}
-      {error && <ErrorPage />}
-      {!error && !skeleton && (
-        <ProfilePage profile={profile} username={username} />
-      )}
-    </main>
+      <main>
```

Comment:
Still a main tag here 😅

Reply:
I understand why there is a main tag here, you are moving the navbar outside the main content which is correct. Can you take a screenshot of the outputted html code to make so we can see that it solves the issue and another main tag doesn’t sneak in from any other component.